tcms/telemetry/urls.py

```diff
@@ -1,7 +1,9 @@
 from django.conf.urls import url
 
-from tcms.telemetry.views import TestingBreakdownView
+from tcms.telemetry.views import TestingBreakdownView, TestingStatusMatrixView
```

Comment:
import `views` so you don't have to change the import line every time you add a new telemetry report.